string name
---
bool success
string info
